src/zappy_ai_src/ai_strategy_management.py

```diff
@@ -437,21 +441,20 @@ def __unexpectedResponseManagement(self):
         """
         print("RENTRE DANS UNEXPECTED RESPONSE MANAGEMENT")
         response: str = self.__lib.getUnexpectedResponse()
-        if response == "":
+        if response == "" or response != None:
```

Comment:
!= None ? Always true

Reply:
this is just a preventive check, nothing crazy don't worry